qiskit_experiments/library/calibration/rabi.py

```diff
@@ -30,19 +30,21 @@
 class Rabi(BaseExperiment):
```

Comment:
Also `Experiment Options` is missing for this class. I think you also need to update `EFRabi` as well.

Reply:
See here: de0e1ce